File: 백준/Bronze/25304. 영수증/영수증.c
#include <stdio.h>
int main()
{
	int total, tcnt, a, cnt, sum=0;
	scanf("%d", &total);
	scanf("%d", &tcnt);
	
	for(int i=1; i<=tcnt; i++){
		scanf("%d %d", &a, &cnt);
		sum+=a*cnt; 
	}
	
	if(total==sum){
		printf("Yes");
	}
	else printf("No");
	
	return 0;
	
}

File: 백준/Bronze/25304. 영수증/영수증.py
sum = int(input())
cnt = int(input())

total = 0

for i in range(cnt):
    data = input().split()
    a = int(data[0])
    b = int(data[1])
    total += a * b

if total == sum:
    print("Yes")
else:
    print("No")
